### DISCUSSION TITLE
Windows Data Server 2019+ official support for Mountain Duck

### DISCUSSION THREAD

**Discussion by davisskchan** [iterate-ch/cyberduck#14553] [Q&A]
Wondering if Windows Data Server 2019 is officially support.  

And to confirm, can I use Mountain Duck to mount Azure Blob Storage as a network drive in Windows server?

Thanks.

**Comment by tatze96** [comment]
The supported operating system versions are always specified as version x or later. Currently, Windows Server 2016 and all newer versions are supported. This also includes Windows Server 2019.

Regarding Azure Blob Storage:
You can mount Azure Blob Storage. Please refer to our [wiki page](https://docs.cyberduck.io/protocols/azure/) for more information.
